packages/cli/src/UserManagement/UserManagementHelper.ts

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 /* eslint-disable @typescript-eslint/no-non-null-assertion */
 /* eslint-disable import/no-cycle */
+import express = require('express');
 import { IsNull, Not } from 'typeorm';
 import { Db, GenericHelpers, ResponseHelper } from '..';
 import config = require('../../config');
```

Comment:
why do we use `require` instead of `import`?

Reply:
https://github.com/n8n-io/n8n/pull/2836#discussion_r811318429